sidebars.js

```diff
@@ -31,41 +31,86 @@ const sidebars = {
   ourSidebar: [
     'cellular-iot',
     'quickstart',
-    'service-stack-overview',
-    'services',
-    'rest-api',
     {
       type: 'category',
-      label: 'Data Streamer',
+      label: 'Services',
       link: { 
         type: 'generated-index', 
-        title: 'emnify multicloud Data Streamer',
-        slug: 'data-streamer'
+        description: 'emnify offers the following high-level products and services:',
+        slug: 'services'
       },
       items: [
-        'data-streamer/getting-started',
-        'data-streamer/connection-types',
-        'data-streamer/stream-types',
-        'data-streamer/managing-data-streams',
-        'data-streamer/usage',
-        'data-streamer/available-integrations'
-      ],
-    },
-    {
-      type: 'category',
-      label: 'Events',
-      link: {
-        type: 'generated-index', 
-        title: 'emnify system events',
-        slug: '/events'
-      },
-      items: [
-        'events/getting-started',
-        'events/event-types',
-        'events/mno-events',
-        'events/usage'
+        'services/global-iot-sim',
+        'services/global-iot-network',
+        'services/iot-cloud-communication-platform',
+        {
+          type: 'category',
+          label: 'Data Streamer',
+          link: { 
+            type: 'generated-index', 
+            title: 'emnify multicloud Data Streamer',
+            slug: 'data-streamer'
+          },
+          items: [
+            {
+              type: 'doc',
+              label: 'Getting Started',
+              id: 'services/data-streamer/getting-started'
+            },
+            {
+              type: 'doc',
+              label: 'Connection Types',
+              id: 'services/data-streamer/connection-types'
+            },
+            {
+              type: 'doc',
+              label: 'Stream Types',
+              id: 'services/data-streamer/stream-types'
+            },
+            {
+              type: 'doc',
+              label: 'Managing Data Streams',
+              id: 'services/data-streamer/managing-data-streams'
+            },
+            {
+              type: 'doc',
+              label: 'Using the Data Streamer',
+              id: 'services/data-streamer/usage'
+            },
+            {
+              type: 'doc',
+              label: 'Available Integrations',
+              id: 'services/data-streamer/available-integrations'
+            }
+          ],
```

Comment:
It looks like this `items` array didn't rebase correctly 🤔  ([see main](https://github.com/EMnify/product-docs/blob/main/sidebars.js#L45-L52))

That's ok though, we should just update before merging:

```suggestion
      items: [
        'services/data-streamer/getting-started',
        'services/data-streamer/connection-types',
        'services/data-streamer/stream-types',
        'services/data-streamer/managing-data-streams',
        'services/data-streamer/usage',
        'services/data-streamer/available-integrations'
      ],
```